src/components/Calendar/Calendar.cy.tsx

```diff
@@ -2,8 +2,8 @@ import React from "react";
 import Calendar, { CalendarEvent } from "@/components/Calendar/Calendar";
 
 describe("<Calendar />", () => {
-    const today = new Date();
-    const tomorrow = new Date();
+    const today = new Date("2021-04-05");
```

Comment:
can you rename these to be more accurate, e.g. `testDate` and `dayAfterTestDate`?